src/frontend/src/App.js

```diff
@@ -2,10 +2,25 @@ import React, { useRef, useState, useEffect } from "react";
 import Map from "./components/Map";
 import Sidebar from "./components/Sidebar";
 import Buttons from "./components/Buttons";
+import LoadingStatus from "./components/LoadingStatus";
 import * as h3 from "h3-js/legacy";
 import "./App.css";
 
 const API_URL = "http://localhost:8000";
+const PATH_TO_TAXA = '/static/taxa_names.json';
+const BAR_STATUS = {
+  inactive: {loadingStatus: "", color: "#b5b5b5"},
+  generating: {loadingStatus: "Generating", color: "#b5b5b5"},
```

Comment:
I would change this to 'Generating ...', the ... indicates that is an ongoing action.  We could also add something like 'this might take a several seconds'. I would keep the same color schema as the buttons for now, generating in blue in this case.